### ISSUE TITLE
Error Listing Loadlib Members 

### ISSUE THREAD

**Issue by dkelosky** [zowe/zowe-cli-ftp-plugin#81]
If I run a command to list members in a loadlib data set, I get an error:
```
C:\dev\asm\hellwrld>zowex zftp list members "ibmuser.work.loadlib"
Command Error:
Unrecognized file list header:  Name      Size     TTR   Alias-of AC --------- Attributes --------- Amode Rmode
```

However, if I run the same for other data set types, I do not receive an error.  My data set attributes are:
```
General Data                              Current Allocation                
 Management class . . : TSO                Allocated tracks  . : 91         
 Storage class  . . . : TSO                Allocated extents . : 4          
  Volume serial . . . : TSU032                                              
  Device type . . . . : 3390                                                
 Data class . . . . . : DCWRKD            Current Utilization               
  Organization  . . . : PO                 Used pages  . . . . : 1,045      
  Record format . . . : U                  % Utilized  . . . . : 95         
  Record length . . . : 0                                                   
  Block size  . . . . : 32760                                               
  1st extent tracks . : 46                                                  
  Secondary tracks  . : 15                Dates                             
  Data set name type  : LIBRARY            Creation date . . . : 2011/05/17
  Data set encryption : NO                 Referenced date . . : 2021/05/13
  Data set version  . : 2                  Expiration date . . : ***None***
```

**Comment by std4lqi** [comment]
FTP returns some additional information for this dataset. Will look into it soon. Thanks for reporting it!

**Comment by dkelosky** [comment]
hi @std4lqi - sorry to trouble you, is this still on your backlog?  I understand you likely have other things going on; I want to comment on here time-to-time to prevent the issue from becoming stale :-)

**Comment by std4lqi** [comment]
@dkelosky , It's on our list. We found zos-node-accessor need be updated to support this header or more dynamically support the other headers we never saw before. Not sure about when it will be supported, but it's one of those we want to put in the next release. Thanks for your asking and patience!

**Comment by dkelosky** [comment]
Great, thanks for the update!

**Comment by tiantn** [comment]
Hi @dkelosky , this issue was fixed in zftp 1.7.0. Could you help to verify it? Thank you!

**Comment by dkelosky** [comment]
thanks
